ui/README.md

```diff
@@ -0,0 +1,90 @@
+# NumaFlow UI
+
+![Numaflow Image](../docs/images/Numa.svg)
+
+A web-based UI for NumaFlow
+
+The UI has the following features:
+* View running pipelines in your namespace
+* View Vertex and Edge Information of your pipeline
+* View BackPressure and Pending Messages
+* View Container (main/udf) Logs for a given vertex
+
+# Development
+
+This doc explains how to set up a development environment so you can get started with UI development
+
+## Prerequisites
+
+Follow the instructions below to set up your development environment.
+
+### Install requirements
+
+These tools are required for development.
+
+1. [`NumaFlow`](https://github.com/numaproj/numaflow/blob/master/docs/DEVELOPMENT.md): Follow the document to prepare `Dataflow` development environment.
+1. [`node`](https://nodejs.org/en/download/): NodeJS - `brew install node`.
+1. [`yarn`](https://yarnpkg.com/): Package manager - `brew install yarn`
+
+### Create a dev cluster
```

Comment:
Maybe we don't need all the stuffs below, they should be covered in the root development doc.

Reply:
Removed